src/users/users.controller.ts

```diff
@@ -0,0 +1,58 @@
+import express from "express";
+import { ValidatedRequest } from "express-joi-validation";
+import {
+  UserAutoSuggestRequestSchema,
+  UserPatchRequestSchema,
+  UserPostRequestSchema,
+} from "../common/common.types";
+import { isEmpty } from "../common/common.utils";
+import usersService from "./users.service";
+
+class UsersController {
+  getUsers(
+    req: ValidatedRequest<UserAutoSuggestRequestSchema>,
+    res: express.Response
+  ) {
+    const login = isEmpty(req.query) ? "" : String(req.query.login);
+    const limit = isEmpty(req.query)
```

Comment:
To be honest, I would avoid this kind of code writing. Shortening expressions doesn't mean perfect always. Generally speaking, we work in a team most of the time. And we code not for ourselves but for everybody. So writing readable and understandable code at first glance is a good practice.